syntax = "proto3";

package checkConnection;

service checkConnection {
    rpc CheckConnection (checkMessage) returns (checkReply);
}

message checkMessage {
    string message=1;
}

message checkReply {
    string message=1;
}